README.md

```diff
@@ -15,17 +18,26 @@ The structure of the repo is as follows:
 | definitions/types | custom CWL data types for inputs to tools and workflows |
 | example_data | example input data, input YAML files, and expected output files for testing |
 
-## CWL Documentation
+## Documentation
 
-For all documentation of pipelines, subworkflows and tools as well as additional information regarding test data, continous integration and configuration, please see the GitHub wiki:
+All documentation of CWL pipelines, subworkflows and tools as well as additional information regarding test data, continous integration and configuration, please see the GitHub wiki:
```

Comment:
If we're removing `For` here, we'll need to change other phrasing, too, perhaps like: 
```suggestion
All documentation of CWL pipelines, subworkflows, and tools as well as additional information regarding test data, continous integration, and configuration can be found on the GitHub wiki:
```